src/clientdlg.cpp

```diff
@@ -1193,65 +1167,36 @@ void CClientDlg::OnCLPingTimeWithNumClientsReceived ( CHostAddress InetAddr, int
     ConnectDlg.SetPingTimeAndNumClientsResult ( InetAddr, iPingTime, iNumClients );
 }
 
-void CClientDlg::Connect ( const QString& strSelectedAddress, const QString& strMixerBoardLabel )
+void CClientDlg::OnConnect ( QString strServerName )
```

Comment:
Add documentation: precondition: ServerAddr() must have been set.

Reply:
If it's a private slot (with an `On` prefix it should be a signal handling slot), the precondition _should_ be guaranteed.  Otherwise there's an error in where the signal is raised.  Of course, as it's potentially passed across threads, thread scope would need considering.